.gitignore

```diff
@@ -0,0 +1,3 @@
+# prevent accidental addition of files from local remark runs
+node_modules/
+package-lock.json
```

Comment:
package-lock should be checked in, otherwise it's hard to reproduce what the Action has done because updates to dependencies could change behavior